core/spec/models/spree/concerns/user_address_book_spec.rb

```diff
@@ -11,56 +11,98 @@ module Spree
     let!(:user) { create(:user) }
 
     describe "#save_in_address_book" do
-      context "saving a default address" do
+      context "saving a shipping address" do
```

Comment:
I think we still need to keep the word `default` in the description, as this block `subject` is saving a _default_ shipping address.

Reply:
Done